Sources/Swift/SentryUserFeedbackFormConfiguration.swift

```diff
@@ -0,0 +1,144 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * Settings to control the behavior and appearance of the UI form.
+ */
+@available(iOSApplicationExtension 13.0, *)
+@objc public class SentryUserFeedbackFormConfiguration: NSObject {
```

Comment:
l: I believe this should extend `SentryUserFeedbackThemeConfiguration`

Reply:
I'm going for composition, don't want `SentryUserFeedbackFormConfiguration` to duplicate `SentryUserFeedbackThemeConfiguration`'s properties